src/com/google/javascript/jscomp/GatherModuleMetadata.java

```diff
@@ -213,6 +213,8 @@ public boolean shouldTraverse(NodeTraversal t, Node n, Node parent) {
         case MODULE_BODY:
           currentModule.hasModuleBody = true;
           break;
+        case DYNAMIC_IMPORT:
+          visitDynamicImport(t, n);
```

Comment:
Actually, this is where the break is missing.

Reply:
That makes a lot more sense - fixed!